frontend/simple/views/components/i18n.vue

```diff
@@ -1,11 +1,37 @@
 <template>
-    <span><slot></slot></span>
+    <component :is="tag">
+      {{transformText}}
+    </component>
 </template>
 <script>
-export default{
-  props: {args: [Object, Array]},
-  beforeMount: function () {
-    this.$slots.default[0].text = this.L(`${this.$slots.default[0].text}.text`, this.args || [], {defaultValue: this.$slots.default[0].text})
+export default {
+  props: {
+    args: [Object, Array],
+    tag: {
+      type: String,
+      default: 'span'
+    }
+  },
+  computed: {
+    transformText () {
+      const text = this.L(
+        this.$slots.default[0].text,
+        this.args || {},
+        { defaultValue: this.$slots.default[0].text }
+      )
```

Comment:
Bug analyzis: `beforeMount` doesn't seems to receive `args` at time to return `L()` correctly after an update (this happened on others places too (ex: summary `1 of 5`), was `{sent} of {total}`. 

So, using a computed method looks like the best solution I could found.